gateway-ha/src/main/java/io/trino/gateway/ha/module/HaGatewayProviderModule.java

```diff
@@ -259,4 +277,28 @@ public MonitorConfiguration getMonitorConfiguration()
     {
         return configuration.getMonitor();
     }
+
+    @Provides
+    public JdbcConnectionManager getConnectionManager()
+    {
+        return this.connectionManager;
+    }
+
+    @Provides
+    public ResourceGroupsManager getResourceGroupsManager()
+    {
+        return this.resourceGroupsManager;
+    }
+
+    @Provides
+    public GatewayBackendManager getGatewayBackendManager()
+    {
+        return this.gatewayBackendManager;
+    }
+
+    @Provides
+    public QueryHistoryManager getQueryHistoryManager()
+    {
+        return this.queryHistoryManager;
```

Comment:
`this.` is redundant.

Reply:
yep, but we use `this` for other getters too.